pkg/sypgp/sypgp.go

```diff
@@ -607,11 +611,157 @@ func SearchPubkey(search, keyserverURI, authToken string) error {
 		}
 	}
 
+	keyText, err = reformatMachineReadableOutput(keyText)
+	if err != nil {
+		return fmt.Errorf("could not reformat key output")
+	}
 	fmt.Printf("%v", keyText)
 
 	return nil
 }
 
+// getEncryptionAlgorithmName obtains the algorithm name for key encryption
+func getEncryptionAlgorithmName(n string) (string, error) {
+	algorithmName := ""
+
+	code, err := strconv.ParseInt(n, 10, 64)
+	if err != nil {
+		return "", err
+	}
+	switch code {
+
+	case 1, 2, 3:
+		algorithmName = "RSA"
+	case 16:
+		algorithmName = "Elgamal"
+	case 17:
+		algorithmName = "DSA"
+	case 18:
+		algorithmName = "Elliptic Curve"
+	case 19:
+		algorithmName = "ECDSA"
+	case 20:
+		algorithmName = "Reserved"
+	case 21:
+		algorithmName = "Diffie-Hellman"
+	default:
+		algorithmName = "unknown"
+	}
+	return algorithmName, nil
+}
+
+//function to obtain a date format from linux epoch time
+func date(s string) (string, error) {
+	if s == "" {
+		return "NULL", nil
+	}
+	if s == "none" {
+		return s + "\t\t\t", nil
+	}
+	c, _ := strconv.ParseInt(s, 10, 64)
+	ret := fmt.Sprintf("%v", time.Unix(c, 0))
+
+	return ret, nil
+}
+
+// reformatMachineReadableOutput reformats the key search output that is in machine readable format
+// see the output format in: https://tools.ietf.org/html/draft-shaw-openpgp-hkp-00#section-5.2
+func reformatMachineReadableOutput(keyText string) (string, error) {
+
+	var output = "\n\t\tFINGERPRINT\t\tALGORITHM  SIZE (BITS)\t\t  CREATION DATE\t\t\tEXPIRATION DATE\t\t\tSTATUS\t\t\tNAME/EMAIL" + "\n"
```

Comment:
I'd prefer to use spaces here, instead of tabs. Depending on your `tabs` settings, it could use 4 spaces as a tab, instead of 8.

Reply:
See the comment before :top: thanks @WestleyK